src/gui_egui/editor.rs

```diff
@@ -127,12 +128,18 @@ impl Default for Library {
                 pos: (0.0, 0.0),
                 select: dummy_input.clone(),
                 m_in: vec![dummy_input.clone(), dummy_input.clone()],
+                scale: 1.0,
             }),
             Rc::new(Register {
                 id: "reg".to_string(),
                 pos: (0.0, 0.0),
                 r_in: dummy_input.clone(),
             }),
+            Rc::new(MIPSCLK {
```

Comment:
is this needed?

Reply:
fixed in 1fadfac92d277c0ca8defc5a5deaf9a1c460c578